src/riak_kv_entropy_manager.erl

```diff
@@ -45,6 +45,7 @@
          get_version/0,
          get_partition_version/1,
          get_pending_version/0,
+         get_upgraded/0,
```

Comment:
Out of curiosity, is anyone (else) calling get_upgraded/0, or is this a convenience/util function for field diagnosis?